src/components/BottomBar/GeoPositionPanel.jsx

```diff
@@ -190,16 +208,17 @@ function GeoPositionPanel() {
     setAddedSceneGraphNodes(nodes);
   }
 
-  function selectCoordinate(location, address) {
+  function selectCoordinate(location, address, extent) {
     const lat = location.y;
     const long = location.x;
+    const alt = extent ? calculateAltitude(extent) : altitude * 1000;
     switch (interaction) {
       case Interaction.flyTo: {
-        luaApi?.globebrowsing?.flyToGeo(currentAnchor, lat, long, altitude);
+        luaApi?.globebrowsing?.flyToGeo(currentAnchor, lat, long, alt);
         break;
       }
       case Interaction.jumpTo: {
-        luaApi?.globebrowsing?.goToGeo(currentAnchor, lat, long, altitude);
+        luaApi.globebrowsing.goToGeo(currentAnchor, Number(lat), Number(long), alt);
```

Comment:
Why are the question marks removed here, but kept in the previous case? Remove them? 

And is that `Number(` conversion necessary? Then it should probably also be done in the previous case